test/test_lowlevel.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from pudb.lowlevel import detect_encoding, decode_lines
+from pudb.py3compat import PY3
+
+
+def test_detect_encoding_nocookie():
+    lines = ['Test Проверка']
```

Comment:
For now I think it's quite OK to cover more code with tests to kind of freeze current codebase stage.